sdk/src/main/java/com/sygic/travel/sdk/places/responseWrappers/PlaceDetailedResponse.kt

```diff
@@ -19,6 +20,6 @@ internal class PlaceDetailedResponse : StResponse() {
 
 	inner class Data {
 		@SerializedName("place")
-		var apiPlace: ApiPlaceDetail? = null
+		internal var apiPlace: ApiPlaceDetail? = null
```

Comment:
remove the internal modifiers here

Reply:
But `ApiPlaceDetail` should be used only internally in SDK module.